avm/res/storage/storage-account/main.bicep

```diff
@@ -700,6 +699,11 @@ output location string = storageAccount.location
 @description('All service endpoints of the deployed storage account, Note Standard_LRS and Standard_ZRS accounts only have a blob service endpoint.')
 output serviceEndpoints object = storageAccount.properties.primaryEndpoints
 
+@description('The array of networ interfaces IDs associated with private endpoints.')
+output privateEndpointsNetworkInterfaceIds array = [
+  for (privateEndpoint, index) in (privateEndpoints ?? []): storageAccount_privateEndpoints[index].outputs.networkInterfaceIds
+]
```

Comment:
Just to align with the output that @krbar implemented in already most other modules where the PE implementation was lifted up to latest. Does not require a re-test, but a re-run of the `Set-AVMModule` script to make sure the readme & template are up to date.

Nothing else to add. Great work @msl0 💪 

```suggestion
@description('The private endpoints of the Storage Account.')
output privateEndpoints array = [
  for (pe, i) in (!empty(privateEndpoints) ? array(privateEndpoints) : []): {
    name: storageAccount_privateEndpoints[i].outputs.name
    resourceId: storageAccount_privateEndpoints[i].outputs.resourceId
    groupId: storageAccount_privateEndpoints[i].outputs.groupId
    customDnsConfig: storageAccount_privateEndpoints[i].outputs.customDnsConfig
    networkInterfaceIds: storageAccount_privateEndpoints[i].outputs.networkInterfaceIds
  }
]
```
```suggestion
@description('The array of networ interfaces IDs associated with private endpoints.')
output privateEndpointsNetworkInterfaceIds array = [
  for (privateEndpoint, index) in (privateEndpoints ?? []): storageAccount_privateEndpoints[index].outputs.networkInterfaceIds
]
```
```suggestion
@description('The array of networ interfaces IDs associated with private endpoints.')
output privateEndpointsNetworkInterfaceIds array = [
  for (privateEndpoint, index) in (privateEndpoints ?? []): storageAccount_privateEndpoints[index].outputs.networkInterfaceIds
]
```